### PR TITLE
RSKIP 148 - Comply with specifications of ERC1820

### PR DESCRIPTION
**Pull request by pmprete**
ERC1820 standard defines a universal registry smart contract where any address (contract or regular account) can register which interface it supports and which smart contract is responsible for its implementation.
This is needed to implement ERC777 contracts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,16 +141,17 @@ RSK Improvement Proposals
 | AE       | Adrian Eidelman              | adrian@iovlabs.org |
 | AL       | Angel Lopez                  | angel@iovlabs.org |
 | AM       | Ariel Mendelzon              | amendelzon@iovlabs.org |
-| MC       | Martín Coll                  | mc@iovlabs.org |
+| MC       | Martín Coll                  | |
```
[the remark this answers is not in the document]
Comment (pmprete) [reply README.md]: Those emails are not valid anymore, just tidying up

File: IPs/RSKIP148.md [hunk 2]
```diff
@@ -0,0 +1,905 @@
...
+
+This standard is backward compatible with [ERC165], as both methods MAY be implemented without conflicting with each other.
+
+## Test Cases
+
+Please check the [0xjac/ERC1820] repository for the full test suite.
+
+## Implementation
+
+The implementation is available in the repo: [0xjac/ERC1820].
```
[the remark this answers is not in the document]
Comment (pmprete) [reply IPs/RSKIP148.md]: As the implementation is the same for all networks i wouldn't bother to fork it

**Comment by julianlen** [comment]
It looks really good!

Two things, at first add rskip to index. At second, I'd describe what a "deployment at singleton address" means, and the same for what is a pseudo-introspection.

Delete Reference section if it is empty.

File: IPs/RSKIP148.md [hunk 3]
```diff
@@ -0,0 +1,906 @@
...
+|**Author**     |PMP |
+|**Purpose**    |Usa |
+|**Layer**      |DApp|
+|**Complexity** |1 |
+|**Status**     |Adopted |
+
+
+## Abstract
+
+This RSKIP proposes complying with the ERC1820 specification, and deployment at he address of the registry on all chains 0x1820a4B7618BdE71Dce8cdc73aAB6C95905faD24 as a standard interface detection.
```
Comment (julianlen) [inline IPs/RSKIP148.md]: just fix this typo

"and deployment at he address.."

Comment (pmprete) [reply IPs/RSKIP148.md]: fixed

**Pull request merged**